python/ray/data/read_api.py

```diff
@@ -698,6 +704,115 @@ def read_parquet(
     )
 
 
+@PublicAPI
+def read_delta(
+    uri: str,
+    delta_package: str = None,
+    *,
+    filesystem: Optional["pyarrow.fs.FileSystem"] = None,
+    columns: Optional[List[str]] = None,
+    parallelism: int = -1,
+    ray_remote_args: Dict[str, Any] = None,
+    tensor_column_schema: Optional[Dict[str, Tuple[np.dtype, Tuple[int, ...]]]] = None,
+    meta_provider: Optional[ParquetMetadataProvider] = None,
+    **arrow_parquet_args,
+) -> Dataset:
+    """Creates a :class:`~ray.data.Dataset` from delta files.
```

Comment:
nit: let's be more explicit `delta files` -> `Databricks Delta files`, and for all other places.

Reply:
It's not just on Databricks, it's for all delta format files 🤔

Reply:
ok, can you add a URL link of delta files? I am sure we are all understanding, but don't want to confuse Ray users, as there're many file formats.